mono/mini/aot-runtime.c

```diff
@@ -4810,34 +4810,14 @@ mono_aot_get_method (MonoDomain *domain, MonoMethod *method, MonoError *error)
 		gboolean volatil = FALSE;
 		MonoMethodSignature *sig;
 
-		/* Same for CompareExchange<T> and Exchange<T> */
 		/* Same for Volatile.Read<T>/Write<T> */
```

Comment:
Maybe similar to do here?

Reply:
Oh yea that looks like dead code.  We already have method bodies like this.

In framework Mono:
https://github.com/mono/mono/blob/73ae9dbc3693201939efce02b2b36c4ce5b352bd/mcs/class/corlib/System.Threading/Volatile.cs#L166-L172

and in netcore Mono:
https://github.com/mono/mono/blob/73ae9dbc3693201939efce02b2b36c4ce5b352bd/netcore/System.Private.CoreLib/shared/System/Threading/Volatile.cs#L220-L229